Make edits to new Data Plane activation doc (#555)
astronomer/docs: astro/data-plane-activation.md

id: 'data-plane-activation'

> description: Prepare for the activation of your data plane

Reviewer: ```suggestion
description: Prepare for the activation of your Astro Data Plane.
```

---

## Overview

> This document provides information about what to expect and prepare for when onboarding to Astro.

Reviewer: ```suggestion
This document provides information about what to expect and prepare for when activating your Data Plane on Astro.
```

> This document provides information about what to expect and prepare for when onboarding to Astro.

Reviewer: ```suggestion
Welcome to Astro! Astro is a modern data orchestration platform, powered by Apache Airflow, that enables your data team to build, run, and observe data pipelines. Astro includes a single-tenant Data Plane in your cloud and a multi-tenant Control Plane in Astronomer's cloud.

This document provides information about what to expect and prepare for when onboarding with our team.
```

> 
> Astro is a modern data orchestration platform, powered by Apache Airflow, that enables your data team to build, run, and observe data pipelines. Astro includes a single-tenant Data Plane in your cloud and a multi-tenant Control Plane in Astronomer's cloud.

Reviewer: ```suggestion
```

> Astro is a modern data orchestration platform, powered by Apache Airflow, that enables your data team to build, run, and observe data pipelines. The architecture of Astro includes a single-tenant Data Plane in your cloud and a multi-tenant Control Plane in Astronomer’s cloud.

Reviewer: That's fine - I moved out the `Powered by Apache Airflow` to make the first statement stronger with a larger emphasis on the value-add (build/run/observe) and then insert the Airflow piece more strongly as the weight of the second sentence.

Modified suggestion back to `in Astronomer's cloud`.

<div class="text--center">
  <img src="/img/docs/architecture-overview.png" alt="High level overview of Astro's architecture" />
</div>

### What to Expect

We’re excited for you to get started with Astro. To start, you'll need to activate your Data Plane. This is a simple process that allows you to see our modern data orchestration infrastructure in action. Once complete, you'll be able to create Deployments, run tasks, and explore our suite of data observability features.

> The Data Plane activation process is guided by an engineer from Astronomer. You can expect this to take about one hour. By the end of the session, you'll have your first pipeline deployed on Astro.

Reviewer: ```suggestion
When you meet with one of our engineers to activate your Data Plane, you can expect this to take about an hour. By the end of the session, you'll have your first pipeline deployed in your own Astro environment.
```

### What to Bring and Know

> Your Data Plane will be deployed into a clean AWS Account in accordance with [Amazon's recommendation](https://docs.aws.amazon.com/whitepapers/latest/organizing-your-aws-environment/benefits-of-using-multiple-aws-accounts.html) for account segregation. Astronomer takes complete responsibility for the operations of this account as described in the [Shared Responsibility Model](shared-responsibility-model.md).

Reviewer: ```suggestion
Your Data Plane will be deployed into a clean AWS Account in accordance with [Amazon's recommendation](https://docs.aws.amazon.com/whitepapers/latest/organizing-your-aws-environment/benefits-of-using-multiple-aws-accounts.html) for account segregation. With our fully managed service, our team takes complete responsibility for the operations of this account as described in the [Shared Responsibility Model](shared-responsibility-model.md).
```

> This model helps the Astronomer team get you started quickly and provides you with cloud-grade reliability and seamless connection to all of your data services.  If you decide to not proceed with Astro, this AWS account and all infrastructure running within it can be deleted at any time.

Reviewer: ```suggestion
This model enables us to get you started quickly and provides you with cloud-grade reliability and seamless connection to all of your data services.  If you decide to not proceed with Astro, this AWS account and all infrastructure running within it can be deleted in its entirety.
```

> Your data plane will be deployed into a clean, dedicated AWS Account, in accordance with [Amazon’s recommendation](https://docs.aws.amazon.com/whitepapers/latest/organizing-your-aws-environment/benefits-of-using-multiple-aws-accounts.html) for account segregation. Astronomer takes complete responsibility for the operations of this account as described in the [Shared Responsibility Model](shared-responsibility-model.md).
> 
> This model helps the Astronomer team to get you started quickly, providing cloud-grade reliability and seamless connection to all of your data services. If you decide not to proceed with Astro, this account can be fully deleted at any point.
> 
> By default, the Astronomer account has no access to your data services. We’ll guide you through how to make these connections securely, whether through peering VPCs or making direct connections.

Reviewer: ```suggestion
Your Data Plane will be deployed into a dedicated and otherwise empty AWS Account in accordance with [Amazon’s recommendation](https://docs.aws.amazon.com/whitepapers/latest/organizing-your-aws-environment/benefits-of-using-multiple-aws-accounts.html) for account segregation. Astronomer takes complete responsibility for the operations of this account as described in the [Shared Responsibility Model](shared-responsibility-model.md). This model helps the Astronomer team get you started quickly and provides you with cloud-grade reliability and seamless connection to all of your data services.

By default, your AWS account for the Astro Data Plane has no direct access to other data services in your network. We’ll guide in making these connections securely either through a VPC peering process or a direct access method. If you decide to not proceed with Astro, this AWS account and all infrastructure running within it can be deleted at any time.
```

Reviewer: @astrorfox Context on my suggestions here:

- Capitalize Data Plane
- Removed dedicated and standardized on `clean` AWS account, per your comments below
- Modified "providing you.." with activate voice "provides you with cloud-grade reliability...". Makes the statement and value-prop stronger.
- Replaced `the Astronomer account` to `AWS account" for accuracy -- the Astronomer account is a different entity.
- Clarified "this AWS account and all infrastructure running within it" because it seems like the thing the user would care about here is the infra getting deleted and not costing them $$.

### Pre-Flight Checklist

> When you arrive at your data plane activation appointment, please ensure that you have:

Author: ```suggestion
Before your data plane activation appointment, please ensure that you have:
```

Reviewer 2: Hmm, the intent of framing as "When you arrive" was to lower the perceived effort and communicate that it can be done a minute before with no problem. Is this stylistic preference, or is there a problem with this framing?

Reviewer: I think we can do `when` here, though I'd adjust to:

> When you meet with an Astronomer engineer to activate your Data Plane, ensure that you have:

- Capitalize Data Plane
- Do we want to say "appointment"? Feel like we don't use that word

Reviewer 2: Feels like there's a tension between a transactional framing and a high-touch one. For this audience (the DevOps engineer), we leaned toward the transactional side intentionally.

Reviewer: Don't feel strongly enough to push back! Let's keep originally wording then, with `Data Plane` capitalized.

> - [Astro CLI](install-cli.md) installed for any users who will develop pipelines.

Author: Tried the checkboxes and they don't work in Markdown, think it's better to just use bullets here.

Reviewer 2: Intent was not necessarily for them to be interactive, but to communicate that it is work to be done (and aligning visually to "Pre-Flight Checklist")

Reviewer: Hm checkboxes would be an interesting feature to explore. If they look good even if they aren't interactive (agree they don't need to be), keeping them is fine with me. Will defer to you here, @jwitz.

> - [Astro CLI](install-cli.md) installed for any users who will develop pipelines.

Reviewer: ```suggestion
- The [Astro CLI](install-cli.md) installed for any users who will develop pipelines.
```

Reviewer 2: Testing this out a bit...would we say "the Astro Runtime" equivalently? Normally with branded objects your goal is to make them standalone nouns.

Reviewer: No, we refer to `Astro Runtime` without the article. I actually agree that our CLI should be a proper noun, but I'd lean on describing the proper noun as the binary itself -- so `install astrocloud` and `astrocloud cli is...`. To my knowledge, it's standard to refer to the more general term for a CLI with an article. This is what these folks do:

- MongoDB: https://www.mongodb.com/docs/mongocli/stable/#what-is-mongocli-
- AWS: https://aws.amazon.com/cli/
- Stripe: https://stripe.com/docs/cli

For whatever it's worth, this also matches current verbal usage by customers & team members

> - A clean AWS Account where you have `CreateRole` permissions.

Reviewer: ```suggestion
- A clean AWS Account where you have `CreateRole` permissions.
```

@astrorfox Is `clean AWS account` the industry standard? Cool if so, but wondering if "empty" or "dedicated" would be clearer

Reviewer 2: There's not really an industry standard here, but `clean` is the most clear word we've found so far. For clarity, `empty` implies resources vs. configuration, and `dedicated` doesn't indicate the state of the account. When/if we find a better word, we can update.

Reviewer: That's fine, I'm cool to standardize on `clean`. Added it to our Style Guide: https://www.notion.so/astronomerio/Astronomer-Industry-Terms-Style-Guide-5d2389d462734e149d2e66a5cf606228#a476e5f8d29842c09f4a523b9c8a9161

> [] A clean AWS Account where you have `CreateRole` permissions.

Reviewer: ```suggestion
[] A clean AWS Account
[] A user with the following permissions on that account:
   - `cloudformation:*`
   - `GetRole`
   - `GetRolePolicy`
   - `CreateRole`
   - `DeleteRolePolicy`
   - `PutRolePolicy`
   - `ListRoles`
   - `UpdateAssumeRolePolicy`
```

- A desired AWS region for your Astro Cluster selected from the list of [supported regions](resource-reference-aws.md#aws-region).
- _If peering VPCs_, preferred subnet CIDR range identified (no smaller than a `/19` range).

> 

Reviewer: @astrorfox Any reason to note link out to our install AWS doc? Understand the different intent between these two docs, but feels weird to not interlink them

Reviewer 2: What does that doc cover that's missing here from the view of getting the trial started?

Reviewer: It doesn't require more information, but we should give users a path to knowing more if they want to. The install on AWS doc includes:

- Quotas
- Note about status page
- Links to AWS documentation
- Note on using AWS Organizations to unify billing

It's more comprehensive. Not that we want to add that info here, but we shouldn't isolate the user from that information either

> [] A desired AWS region for your Astro Cluster selected from the list of [supported regions](resource-reference-aws.md#aws-region).

Reviewer: ```suggestion
[] A desired AWS region for your Astro Cluster, selected from [our list of supported regions](resource-reference-aws.md#aws-region).
```

> [] _If peering VPCs_, preferred subnet CIDR range identified (no smaller than a `/19` range).

Reviewer: ```suggestion
[] _If peering VPCs_, preferred subnet CIDR range identified. This should be no smaller than a `/19` range.
```

### What’s Next

> After your data plane is activated, you’ll be able to spin up new Airflow environments, deploy pipelines through the Astro CLI, and witness the ability of Astro Runtime.
> 
> Our engineers want to help you get started quickly and get back to focusing on your pipelines. We’ll reach out in a few days to see how you’re doing, but don’t hesitate to reach out in the interim via Slack or e-mail.

Reviewer: ```suggestion
Once your Data Plane is activated, you’ll be able to create new Deployments, deploy DAGs via the Astro CLI, and experience the differentiated Apache Airflow experience that is powered by Astro Runtime.

Our team at Astronomer wants to help you get started with Astro as quickly as possible and let your team focus their efforts on developing, testing, and running data pipelines. We’ll keep in touch following the Data Plane activation meeting to see how you’re doing, but don’t hesitate to reach out to us via Slack or email. We're here to help.
```

`Witness the ability of Astro Runtime` is weird wording to me & and I don't feel like `witness` is the most appropriate or strongest term here. Can we be a little more grounded here?

Made a few copy suggestions here.

> Our team at Astronomer wants to help you get started with Astro as quickly as possible and let your team focus their efforts on developing, testing, and running data pipelines. We’ll keep in touch following the Data Plane activation meeting to see how you’re doing, but don’t hesitate to reach out to us via Slack or email. We're here to help.

Reviewer: ```suggestion
Our team at Astronomer is committed to helping you get started with Astro as quickly as possible and let your team focus their efforts on developing, testing, and running data pipelines. We’ll keep in touch following the Data Plane activation meeting to see how you’re doing, but don’t hesitate to reach out to us via Slack or email!
```
